### ISSUE TITLE
[BUG Tracker] /forums/<thread_id>/create does not work

### ISSUE THREAD

**Issue by stuart909** [Pengin-Open-Source/pengin-pi#261]
**Describe the bug**
I was able to create generic threads on the new site using our EC2 server and MySQL, however, the threads will not load properly.  We need to make sure our threads are correctly rendering with MySQL.

- [ ] #284
- [ ] #285
- [ ] #286
- [ ] #287

**Comment by stuart909** [comment]
I have confirmed that the thread name and ID is in the database.  The page just refuses to populate the threads onto the view.

**Comment by stuart909** [comment]
This is a permissions issue.  Admin has the right to create a thread, and does so successfully.  The issue here is that we are not setting Role ID to the thread, and my admin user did not have the correct role for the thread.  The code checks role, and in this case, it was None.  I then set UserRole user id to the thread's role_id.  I still couldn't render the threads.  I then added role_id user and my user id to UserRoles and then I could render.  

To prevent this issue, I will go ahead and add the ability for the admin to set which UserRole they want to assign to the table.  That will bind the user role into the thread.  The user creation system, by default, will create the UserRole entry for user.  So any thread with the user role ID will render for validated users without a need for fixing.
